_maps/map_files/LimaStation/LimaStation.dmm

```diff
@@ -23835,7 +23835,6 @@
 	dir = 4;
 	pixel_x = -32
 	},
-/obj/structure/displaycase/curator,
```

Comment:
Is this intended?

Reply:
Seems not, don't see it in automaps, unsure why this happened